h2o-r/h2o-package/R/models.R

```diff
@@ -838,6 +838,9 @@ h2o.staged_predict_proba <- staged_predict_proba.H2OModel
 #'        desired
 #' @param newdata An H2OFrame object in which to look for
 #'        variables with which to predict.
+#' @param output_format Specify how to output feature contributions in XGBoost - XGBoost by default outputs
+#'                      contributions for 1-hot encoded features, specifying a 'Compact' output format will produce
+#'                      a per-feature contribution. One of: 'Original', 'Compact' (default: 'Original').
```

Comment:
Defaults in R are usually specified as sentence, e.g., `Defaults to Original.`  and the "One of" part is redundant since this is already present in the function declaration.

Reply:
Changed, thx